dev_guide/builds/build_environment.adoc

```diff
@@ -37,14 +43,17 @@ env:
         fieldPath: metadata.name <3>
 ----
 
-<1> the value should be resolved using the following references
-<2> the following `fieldPath` refers to a jsonpath in the `build`
-<3> the jsonpath to the field in the `build`
+<1> `valueFrom` should be resolved using `fieldPath` and `fieldRef`.
+<2> `fieldPath` refers to a `jsonpath` in the `build`.
+<3> `jsonpath` to the field in the `build`.
+
```

Comment:
Hmmm, how about we instead follow the guidance of the Downward API page environment variable examples (this one for instance: https://docs.openshift.org/latest/dev_guide/downward_api.html#dapi-configmaps-using-environment-variables) and just get rid of the footnote things?  I can whip up a sample to put here if everyone agrees.